projects/admin/src/app/document-editor/document-editor.component.ts

```diff
@@ -1,17 +1,13 @@
-import { Component, OnInit, Inject } from '@angular/core';
```

Comment:
DocumentEditorComponent seems not to be used anymore. Can we remove it completely?